summary_workflows/quantification/specification/Q_benchmark_specification.md

```diff
@@ -30,8 +37,26 @@ To compute the metrics, the poly(A) sites identified from the RNAseq data by the
 - find multiple predicted sites overlapping one ground truth site
   - weigths are added for the predicted sites based on distance to the ground truth site
   - expression can be calculated by `sum(weight_i*expression_i)` to get a single value matching the one ground truth site OR just summed without considering weight
+- find ground truth sites that do not have any identified sites mapped to them - the expression of identified sites is set to zero for those sites
+- find predicted sites that were not mapped to any ground truth sites
+  
+The correlation is calculated based on ground truth and identified PAS that overlap for a given window size, as well as ground truth PAS that do not have any identified sites mapped to them.
+
+2. Total expression of non-matched sites
+
+The expression values of all predicted sites that were not mapped to any ground truth site are summed together. As the PAS expression levels are provided as TPM, these values are relative to the expression of all identified PAS.
+  
+3. Mean Squared Error (MSE) of relative PAS usage calculated from RNAseq-based PAS quantification and orthogonal 3'end seq data
+
+The poly(A) sites identified by the benchmarked tool are mapped to ground-truth PAS in the orthogonal dataset as in point 1.
+The identified PAS are then assigned to genes based on genome annotation, and relative PAS usage is calculated separately for each gene.
+PAS that do cannot be assigned to any genes are discarded.
+The MSE is then calculated globally for all poly(A) sites assigned to genes.
```

Comment:
In the ground truth there is no restriction on the location of poly(A) sites within a gene.

However, is an MSE metric necessary? Isn't a correlation of PAS usage more fine-grained?

Reply:
@mzavolan I don't think it has to be MSE, but I think it's useful to have a metric that includes relative usage and not only TPM, as TPM would be impacted if gene expression is over- or underestimated in RNAseq compared to 3'end seq. Do you think it would be better to use MSE for relative usage or to remove relative usage metrics altogether?

@dominikburri 
> calculated as the fraction of most proximal PAS of all PAS

I'm not sure I understand what you mean here, so I'll explain what I had in mind: 1. find all PAS for a given gene, 2. sum TPM values of all PAS, 3. calculate fraction for each PAS by dividing TPM_PAS by TPM_sum. By "all PAS for a given gene" I mean all ground truth PAS for a given gene or all predicted PAS that are matched to ground truth for a given gene.

> Given the above definition, the MSE would measure whether the participant can correctly identify relative PAS usage (or alternative polyadenylation). This is normalised and does not depend on absolute quantification levels, the correlation metric deals with this.

Yes.

> And with gene you mean exonic locations only? Or would you include intronic regions as well for consideration?

If ground truth includes intronic PAS, then predicted PAS in introns will also be counted.

Reply:
Hi @daneckaw, thank you for the explanations! It is clear for me now.

Reply:
Thanks for clarification  @mzavolan. I've changed the metric to Correlation instead of MSE.